ci/code_checks.sh

```diff
@@ -126,6 +126,10 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     invgrep -r -E --include '*.py' "[[:space:]] pytest.raises" pandas/tests/
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    MSG='Check for python2-style file encodings' ; echo $MSG
+    invgrep -R --include="*.{py,pyx}" -E "# -\*- coding: utf-8 -\*-" pandas scripts
```

Comment:
Does the `--include` work with quotes as written? It's not detecting instances of the pattern for me when the quotes are present, but without the quotes, `--include=*.{py,pyx}`, it is detecting the pattern.  I'm stuck on the windows subsytem for linux at the moment though, so it could just be some weirdness with that.

Reply:
Yea this doesn't work for me. Not sure how grep handles the curly braces inside of quotes but I think that is problematic. Either of the below work for me (similar to yours with escaping of asterisk)

```sh
--include="*."{py,pyx}
--include=\*.{py,pyx}
```

Reply:
Ok, I´m on windows, so couldn`t check. I can change it the next time I get to the computer.

Reply:
I've changed that line. Could someone check if it works properly on pyx files?

Reply:
Worked for me!

Reply:
Uh, I just changed it again to be the same style as the rest of code_checks.sh. So it's now of the style ``--include="*.py" --include="*.pyx"`` style, which is perhaps more readable/less pearl-ish :-)

Reply:
Works as well